app/src/main/java/com/lehtimaeki/askold/utils/GameTileResources.kt

```diff
@@ -4,152 +4,146 @@ import com.lehtimaeki.askold.R
 
 object GameTileResources {
 
-    val NUMBERS = listOf(
-        R.drawable.number_0,
-        R.drawable.number_1,
-        R.drawable.number_2,
-        R.drawable.number_3,
-        R.drawable.number_4,
-        R.drawable.number_6,
-        R.drawable.number_6,
-        R.drawable.number_7,
-        R.drawable.number_8,
-        R.drawable.number_9,
-        R.drawable.number_10,
-        R.drawable.number_11,
-        R.drawable.number_12,
-        R.drawable.number_13,
+    val NUMBERS = listOf<Int>(
+        R.drawable.ic_number_0,
+        R.drawable.ic_number_1,
+        R.drawable.ic_number_2,
+        R.drawable.ic_number_3,
+        R.drawable.ic_number_4,
+        R.drawable.ic_number_5,
+        R.drawable.ic_number_6,
+        R.drawable.ic_number_7,
+        R.drawable.icnumber8,
+        R.drawable.icnumber9,
+        R.drawable.icnumber10,
+        R.drawable.icnumber11,
+        R.drawable.icnumber12,
+        R.drawable.icnumber13,
         R.drawable.number_14,
         R.drawable.number_15,
         R.drawable.number_16,
         R.drawable.number_17,
         R.drawable.number_18,
         R.drawable.number_19,
-        R.drawable.number_20,
-        )
-
+        R.drawable.number_20
+    )
```

Comment:
I wonder why these have 3 different naming schemes...

Reply:
There were a lot of errors with these vectors, because for no reason, AS didn’t see the imported svgs anymore. So I imported some of them several times with a different name